specification/maps/data-plane/Search/preview/2022-09-01-preview/search.json

```diff
@@ -0,0 +1,966 @@
+        },
+        "batchItems": {
+          "description": "Array containing the batch results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GeocodingBatchResponseItem"
+          }
+        }
+      }
+    },
+    "Features": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/FeaturesItem"
+      }
+    },
+    "FeaturesItem": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "must be Feature",
+          "enum": [
+            "Feature"
+          ],
+          "x-ms-enum": {
+            "name": "FeaturesItemTypeEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Feature"
+              }
+            ]
+          }
+        },
+        "id": {
+          "type": "string",
+          "description": "ID for feature returned"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "One of: \n* Address\n* RoadBlock\n* RoadIntersection",
+              "readOnly": true
+            },
+            "confidence": {
+              "$ref": "#/definitions/Confidence"
+            },
+            "matchCodes": {
+              "$ref": "#/definitions/MatchCodes"
+            },
+            "address": {
+              "$ref": "#/definitions/Address"
+            },
+            "geocodePoints": {
+              "$ref": "#/definitions/GeocodePoints"
+            }
+          }
+        },
+        "geometry": {
+          "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/definitions/GeoJsonPoint"
+        },
+        "bbox": {
+          "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/definitions/BoundingBox"
+        }
+      },
+      "required": [
+        "geometry"
+      ]
+    },
+    "GeocodingResponse": {
+      "description": "This object is returned from a successful Geocoding call",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/FeatureType"
+        },
+        "features": {
+          "$ref": "#/definitions/Features"
+        },
+        "nextLink": {
+          "$ref": "#/definitions/NextLink"
+        }
+      }
+    },
+    "Address": {
+      "description": "The address of the result",
+      "type": "object",
+      "properties": {
+        "addressLine": {
+          "description": "AddressLine that includes Street Name and Number",
+          "type": "string",
+          "readOnly": true
+        },
+        "locality": {
+          "description": "locality property",
+          "type": "string",
+          "readOnly": true
+        },
+        "neighborhood": {
+          "description": "neighborhood property",
+          "type": "string",
+          "readOnly": true
+        },
```

Comment:
Recommend to remove `readOnly: true` if this schema is not used in a request-body.

Reply:
updated